pkg/virt-api/webhooks/validating-webhook/admitters/vmi-update-admitter_test.go

```diff
@@ -589,6 +593,16 @@ var _ = Describe("Validating VMIUpdate Admitter", func() {
 		return res
 	}
 
+	makeDisksWithIOThreadsAndBus := func(bus v1.DiskBus, indexes ...int) []v1.Disk {
+		res := makeDisksWithBus(bus, indexes...)
+		for i, index := range indexes {
+			if i == len(indexes)-1 {
+				res[index].DedicatedIOThread = pointer.P(true)
+			}
+		}
```

Comment:
I might be missing something but why do you need the for?
you basically just use the last index and update the res of that index.. so why not?:
if len(indexes) > 0 {
		lastIndex := indexes[len(indexes)-1]
		res[lastIndex].DedicatedIOThread = pointer.P(true)
}